pulp_python/app/models.py

```diff
@@ -116,6 +116,9 @@ class PythonPackageContent(Content):
     obsoletes_dist = JSONField(default=list)
     requires_external = JSONField(default=list)
     classifiers = JSONField(default=list)
+    project_urls = JSONField(default=dict)
+    description_content_type = models.TextField()
+    provides_extra = models.TextField()
```

Comment:
I'm not 100% sure because the example [here](https://packaging.python.org/specifications/core-metadata/#id29) doesn't show it, but I believe this can be specified multiple times.  That's my interpretation of "multiple-use"

Reply:
Have you seen any in-the-wild examples?

Reply:
No, I haven't found any. This field doesn't seem to be exposed on PyPI, so I have no idea.